---a/net/minecraft/entity/ai/EntityAISwimming.java
+++b/net/minecraft/entity/ai/EntityAISwimming.java
@@ -3,29 +3,25 @@
 import net.minecraft.entity.EntityLiving;
 import net.minecraft.pathfinding.PathNavigateGround;
 
-public class EntityAISwimming extends EntityAIBase
-{
+public class EntityAISwimming extends EntityAIBase {
+
     private EntityLiving theEntity;
     private static final String __OBFID = "CL_00001584";
 
-    public EntityAISwimming(EntityLiving p_i1624_1_)
-    {
+    public EntityAISwimming(EntityLiving p_i1624_1_) {
         this.theEntity = p_i1624_1_;
         this.setMutexBits(4);
-        ((PathNavigateGround)p_i1624_1_.getNavigator()).func_179693_d(true);
+        ((PathNavigateGround) p_i1624_1_.getNavigator()).func_179693_d(true);
     }
 
-    public boolean shouldExecute()
-    {
+    public boolean shouldExecute() {
         return this.theEntity.isInWater() || this.theEntity.isInLava();
     }
 
-    public void updateTask()
-    {
-        if (this.theEntity.getRNG().nextFloat() < 0.8F)
-        {
+    public void updateTask() {
+        if (this.theEntity.getRNG().nextFloat() < 0.8F) {
             this.theEntity.getJumpHelper().setJumping();
         }
     }
-}
+}
